includes/module/settings/CredentialsSettings.php

```diff
@@ -50,12 +50,12 @@ public function generateForm()
             array(
                 'col' => 4,
                 'type' => 'switch',
-                'label' => $this->module->l('Sandbox Mode'),
+                'label' => $this->module->l('Production'),
                 'name' => 'MERCADOPAGO_SANDBOX_STATUS',
                 'is_bool' => true,
-                'desc' => $this->module->l('Choose "YES" to test your store before selling. ') .
-                    $this->module->l('Switch to "NO" to disable test mode ') .
-                    $this->module->l('and start receiving online payments.'),
+                'desc' => $this->module->l('Select "YES" only when you are ready to sell. ') .
```

Comment:
Essas quebras de linha não podem gerar problema? (para todos os pontos do código que usam isso)
Tenho que traduzir exatamente "Select "YES" only when you are ready to sell.". Deveria ser a frase completa, não?

Reply:
A função de tradução tem limite de caracteres, então tem que ser feita essa quebra de linha. E fazendo isso não gera nenhum problema.